File: pset1/cash/cash.c
#include <stdio.h>
#include <cs50.h>
#include <math.h>

int main(void)
{
    float dollars;
    do
    {
        dollars = get_float("Change owed: ");
    }
    while (dollars < 0.0);
    
    //converts dollars to an integer of cents
    int cents = round(dollars * 100);
    
    int numCoins = 0;

    //finds number of quaters
    numCoins = numCoins + (cents / 25);
    cents = cents % 25;
    
    //finds number of dimes
    numCoins = numCoins + (cents / 10);
    cents = cents % 10;
    
    //finds number of nickels
    numCoins = numCoins + (cents / 5);
    cents = cents % 5;
        
    //finds number or pennies
    numCoins = numCoins + (cents / 1);
    
    printf("%i\n", numCoins);
}

File: pset6/cash/cash.py
from cs50 import get_float

numCoins = 0
change = get_float("Change owed: ")

while change < 0:
    change = get_float("Change owed: ")

cents = int(change * 100)

# print(f"{cents}")

numCoins = numCoins + (cents // 25)
cents = cents % 25

# print(f"{numCoins}")

numCoins = numCoins + (cents // 10)
cents = cents % 10

numCoins = numCoins + (cents // 5)
cents = cents % 5

numCoins = numCoins + (cents // 1)

print(f"{numCoins}")